packages/playground/website/public/logger.php

```diff
@@ -40,52 +40,27 @@ function response($ok, $error = null)
     die(json_encode($response_data));
 }
 
-/**
- * Validate the message format
- *
- * @param string $message - The message to validate
- * @return bool - If the message is valid
- */
-function validate_message($message)
-{
-    // Validate description. Description is required
-    preg_match('/(?<=What happened\?\n\n)(.*?)(?=\n\nLogs)/s', $message, $description);
-    if (empty($description)) {
-        return false;
-    }
-
-    // Validate logs if exists. Logs need to match the PHP error log format
-    preg_match('/(?<=Logs\n\n)(.*?)(?=\n\nUrl)/s', $message, $logs);
-    if (!empty($logs)) {
-        $logs = $logs[0];
-        if (preg_match('/\[\d{2}-[A-Za-z]{3}-\d{4} \d{2}:\d{2}:\d{2} UTC\](.*)/s', $logs) !== 1) {
-            return false;
-        }
-    }
-
-    // Validate URL if exists
-    preg_match('/(?<=Url\n\n)(.*)/s', $message, $url);
-    if (!empty($url)) {
-        $url = $url[0];
-        if (filter_var($url, FILTER_VALIDATE_URL) === false) {
-            return false;
-        }
-    }
-
-    return true;
-}
-
 if (empty($token)) {
     response(false, 'No token provided');
 }
 
-if (!isset($_POST['message'])) {
-    response(false, 'No message provided');
+if (!isset($_POST['description'])) {
+    response(false, 'No description provided');
 }
-$text = $_POST['message'];
+$text = "What happened?\n\n" . $_POST['description'];
 
-if (!validate_message($text)) {
-    response(false, 'Invalid message');
+if (isset($_POST['logs'])) {
+    if (preg_match('/\[\d{2}-[A-Za-z]{3}-\d{4} \d{2}:\d{2}:\d{2} UTC\](.*)/s', $_POST['logs']) !== 1) {
+        response(false, 'Logs do not match the PHP error log format');
```

Comment:
I'm a bit skeptical about this check – I think we'll get surprised with some false positives eventually and also this  only for a single substring. What would be a scenario it protects against?

Reply:
The only thing it does is ensure the format is correct (if the regex works as expected), but it doesn't protect us.
I will remove it and just check if it's empty.

Reply:
Updated [74f6c7a](https://github.com/WordPress/wordpress-playground/pull/1102/commits/74f6c7a6907d7211e3c5cbacf933ec2d32dc5c4a)